newIDE/app/public/external/jfxr/jfxr-main.js

```diff
@@ -80,5 +80,7 @@ ipcRenderer.on('jfxr-open', (event, receivedOptions) => {
     initialResourcePath: receivedOptions.resourcePath,
     extension: '.wav',
     headerStyle
-  })
-})
+  });
+  // disable google analytics from collecting personal information, as requested by wend1go :)
```

Comment:
Let's not put nicknames or names in the codebase, the rest of the comment is sufficient by itself :)
Also nitpicking: Capital D to disable

Reply:
amended :)